test/policy-collection/policy_gatekeeper_operator_test.go

```diff
@@ -544,11 +544,11 @@ var _ = Describe("", func() {
 				out, _ := utils.KubectlWithOutput("get", "ns", "openshift-gatekeeper-system", "--kubeconfig="+kubeconfigManaged)
 				return out
 			}, defaultTimeoutSeconds*4, 1).Should(ContainSubstring("namespaces \"openshift-gatekeeper-system\" not found"))
```

Comment:
Does this need to all be `gatekeeper-system` also or is this something else?

Reply:
This is the namespace where the gatekeeper-operator installed the "actual" gatekeeper pods. But, since noticing this and running some more tests on the ROSA cluster, it looks like the upstream gatekeeper-operator will not work on ROSA clusters. The namespace change makes it able to be installed, but not actually function (since it will still try to create this `openshift-gatekeeper-system` namespace and be denied).

I'm not sure exactly why, but the downstream gatekeeper operator *does* have permissions to make the `openshift-gatekeeper-system` namespace, so it's unaffected by this and can still run correctly on ROSA.